spec/fixtures/factories.rb

```diff
@@ -842,4 +842,14 @@
       end
     end
   end
+
+  factory :export_health_certificate, parent: :document do
+    base_path { "/export-health-certificates/example-document" }
+    document_type { "export_health_certificate" }
+  end
+
+  factory :traffic_commissioner_regulatory_decision, parent: :document do
+    base_path { "/traffic_commissioner_regulatory_decisions/example-document" }
+    document_type { "traffic_commissioner_regulatory_decision" }
+  end
```

Comment:
Adding these factories could be in their own separate commit (prior to adding the general tests), highlighting that these should have been added when they were first defined but that that step got missed).